src/containers/ClusterModeGuard/ClusterModeGuard.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 
-import {useTypedSelector} from '../../lib';
+import {useTypedSelector} from '../../utils/hooks';
```

Comment:
Updated such imports as well to prevent circular deps when embedded-ui is used as package